frontend/utility/platform-windows.cpp

```diff
@@ -15,30 +15,29 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ******************************************************************************/
 
-#include <algorithm>
-#include <sstream>
-#include "obs-config.h"
-#include "obs-app.hpp"
+#define WIN32_LEAN_AND_MEAN
+
 #include "platform.hpp"
 
-#include <qt-wrappers.hpp>
+#include <OBSApp.hpp>
+
+#include <util/windows/ComPtr.hpp>
+#include <util/windows/WinHandle.hpp>
 #include <util/windows/win-version.h>
-#include <util/platform.h>
 
-#define WIN32_LEAN_AND_MEAN
```

Comment:
See other comments about `WIN32_LEAN_AND_MEAN`. Marking in case there is a further change to make.

Reply:
See above.